WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -81,14 +79,14 @@ class GutenbergSettings {
     }
 
     private func setGutenbergEnabledForAllSites() {
-        let allBlogs = (try? BlogQuery().blogs(in: context)) ?? []
+        let allBlogs = Environment.current.contextManager.performQuery({ (try? BlogQuery().blogs(in: $0)) ?? [] })
```

Comment:
Did you consider defining a computed `var` for the context manager, to streamline the coding a bit?

```swift
var contextManager: CoreDataStack { Environment.current.contextManager }

// ...

let allBlogs = contextManager.performQuery...

Reply:
That's a good idea! Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20044/commits/7bd6f006b25480fe50eeca2b09bc40814e360477.